### PR TITLE
Updating docs -- WIP.

### PR DESCRIPTION
**Pull request by brianofrokk3r**
Don't merge yet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,42 @@
...
+# MALOS-eye examples
 
-* Demographics example. Detect faces and get age, gender, emotion and pose by querying a remote server.
+### Pre-Requisites
+```
+# Base repository and assoc. packages
+echo "deb http://packages.matrix.one/matrix-creator/ ./" | sudo tee --append /etc/apt/sources.list;
+sudo apt-get update;
+sudo apt-get upgrade;
+sudo apt-get install libzmq3-dev xc3sprog matrix-creator-openocd wiringpi cmake g++ git;
```
Comment (arhuaco) [inline README.md]: Missing matrix-creator-init, malos-eye and matrix-creator-malos from the package list.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,42 @@
...
+### Getting Started with the Examples
+```
+cd examples;
+npm install;
+```
+
+### Face Detection with Demographics
+Demographics example. Detect faces and get age, gender, emotion and pose by querying a remote server.
+```
+query_demographics.js
```
Comment (arhuaco) [inline README.md]: node query_demographics.js

File: README.md [hunk 3]
```diff
@@ -1,8 +1,42 @@
-# Examples for MALOS-eye
+# MALOS-eye examples
 
-* Demographics example. Detect faces and get age, gender, emotion and pose by querying a remote server.
+### Pre-Requisites
+```
+# Base repository and assoc. packages
+echo "deb http://packages.matrix.one/matrix-creator/ ./" | sudo tee --append /etc/apt/sources.list;
+sudo apt-get update;
```
Comment (arhuaco) [inline README.md]: ; is not needed in these 4 lines.

File: README.md (hunk 1 above)
Comment (arhuaco) [reply README.md]: Small note about the need of a restart.

File: README.md [hunk 4]
```diff
@@ -1,11 +1,46 @@
...
+# MALOS-eye examples
 
-* Demographics example. Detect faces and get age, gender, emotion and pose by querying a remote server.
+### Pre-Requisites
+```
+# Base repository and assoc. packages
+echo "deb http://packages.matrix.one/matrix-creator/ ./" | sudo tee --append /etc/apt/sources.list
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt-get install libzmq3-dev xc3sprog matrix-creator-openocd wiringpi cmake g++ git
```
Comment (arhuaco) [inline README.md]: Still missing matrix-creator-init matrix-creator-malos and malos-eye ...

Comment (brianofrokk3r) [reply README.md]: Derp. Thought I grabbed this from the right docs.

**Review by arhuaco — APPROVED** [review]

**Pull request merged**